bin.js

```diff
@@ -3,7 +3,7 @@
 var path = require('path')
 var log = require('npmlog')
 var fs = require('fs')
-var async = require('async')
+var eachSeries = require('each-series-async')
```

Comment:
The similarly-named [async-each-series](https://www.npmjs.com/package/async-each-series) module is significantly more depended upon and appears to provide the same API.

Reply:
I picked `each-series-async` because of its simple code, typically don't care about how many dependents a package has. I'm fine with either though.